Templates/Java/requests_extensions/IBaseEntityCollectionRequest.java.tt

```diff
@@ -87,6 +87,6 @@ import <#=importNamespace#>.http.IHttpRequest;
      *
 	 * @return the updated request
 	 */
-	<#=c.ITypeCollectionRequest()#> skipToken(String skipToken);
+	<#=c.ITypeCollectionRequest()#> skipToken(final String skipToken);
```

Comment:
You are a big advocate of compile-time constraints and I enjoy all these `final` changes. Really appreciate it!

Reply:
the more work I can give to the compiler checking for what clumsy me can do, the better!